qa/tasks/cephfs/test_nfs.py

```diff
@@ -118,3 +136,10 @@ def test_create_multiple_exports(self):
         self._create_export(export_id='4', extra_cmd=[self.pseudo_path+'3', fs_path.strip()])
         self._test_delete_cluster()
         self._check_export_obj_deleted(conf_obj=True)
+
+    def test_exports_on_mgr_restart(self):
+        self._create_default_export()
+        self._unload_module("cephadm")
+        self._load_module("cephadm")
+        self._orch_cmd("set", "backend", "cephadm")
+        self._delete_export()
```

Comment:
How does this test the persistence of exports across restart? _delete_export() is going to succeed both when the export is deleted and when it doesn't exist. You'll want to check the stderr or stdout to confirm.

I suggest you do _create_default_export() again and check if you get the 'export already exists' message. Or even better once you add the 'export_ls' command, use that to check for exports.

Reply:
I will be adding 'export ls' command here after your pr on it gets merged. Locally while testing, I have found  sometimes mgr on restarting does not reinitialize the volume module class objects and throws rados error. To catch this error I have added _delete_export().